Source/Urho3D/SystemUI/Gizmo.cpp

```diff
@@ -188,7 +188,9 @@ bool Gizmo::Manipulate(const Camera* camera, Node** begin, Node** end)
                 // A workaround for ImGuizmo bug where delta matrix returns absolute scale value.
                 if (!nodeScaleStart_.contains(node))
                     nodeScaleStart_[node] = node->GetScale();
-                node->SetScale(nodeScaleStart_[node] * delta.Scale());
+                node->SetScale(currentOrigin.Scale());
```

Comment:
??

Reply:
This fix is an another PR. I'll roll it back.